src/models/unet_2d.rs

```diff
@@ -103,6 +103,13 @@ impl UNet2DConditionModel {
             .map(|i| {
                 let BlockConfig { out_channels, use_cross_attn, attention_head_dim } =
                     config.blocks[i];
+
+                // Enable automatic attention slicing if the config sliced_attention_size is set to 0.
+                let sliced_attention_size = match config.sliced_attention_size {
+                    Some(s) => if s == 0 { Some(attention_head_dim / 2) } else { None },
```

Comment:
Using `Some(0)` in the pattern matching would feel a bit more idiomatic, any reason not to do so?

Reply:
No reason, updating it!